includes/admin/views/html-admin-page-status-report.php

```diff
@@ -528,6 +528,9 @@
 				if ( ! $page_id ) {
 					echo '<mark class="error">' . __( 'Page not set', 'woocommerce' ) . '</mark>';
 					$error = true;
+				} else if ( get_post_status ( $page_id ) != 'publish' ) { // make sure the page isn't private or a draft
```

Comment:
Remove space after get_post_status and do literal comparison !== Comment can be removed.